app/router.js

```diff
@@ -1,44 +1,37 @@
-/* eslint-disable ember/no-incorrect-calls-with-inline-anonymous-functions */
-
 import EmberRouter from '@ember/routing/router';
-import config from './config/environment';
-
-import { get } from '@ember/object';
 import { inject as service } from '@ember/service';
-import { scheduleOnce } from '@ember/runloop';
+import config from 'ember-guides/config/environment';
 
-const Router = EmberRouter.extend({
-  location: config.locationType,
-  rootURL: config.rootURL,
+export default class Router extends EmberRouter {
```

Comment:
TODO:

After updating `ember-source` to `v3.26`, try injecting the `router` service when calling `routeDidChange`.